src/main/java/org/broadinstitute/hellbender/cmdline/DeprecatedArtifactsRegistry.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.cmdline;
+
+import scala.Tuple2;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * When a tool is removed from GATK (after having been @Deprecated for a suitable period), an entry should
+ * be added to this list to issue a message when the user tries to run that tool.
+ *
+ * NOTE: Picard tools should be listed here as well, since by definition such tools will not be found in
+ * the Picard jar.
+ */
+public class DeprecatedArtifactsRegistry {
+
+    // Mapping from tool name to string describing the major version number where the tool first disappeared and
+    // optional recommended alternatives
+    private static Map<String, Tuple2<String, String>> deprecatedTools = new HashMap<>();
```

Comment:
Use `org.apache.commons.lang3.tuple.Pair` instead of Scala `Tuple2`

Reply:
Done.